src/main/resources/i18n/messages_en.properties

```diff
@@ -234,3 +234,8 @@ email.courseNotification.tutorialGroupUnassignedNotification.text = You were una
 
 email.courseNotification.tutorialGroupDeletedNotification.title = Tutorial group was deleted.
 email.courseNotification.tutorialGroupDeletedNotification.text = Tutorial group "{0}" in "{1}" was deleted.
+
+# Build Agent Pause Email
+email.buildAgent.SelfPaused.title=Build Agent Paused
+email.buildAgent.SelfPaused.text1=The build agent "{0}" paused itself due to consecutive failed or timed out build jobs.
```

Comment:
Would it make sense to include the number of failed / timed-out build jobs? 

Might help users to understand when it is triggered without looking into the coder and easier adjust it in case the setup was done by another admin. 

No strong opinion though, rather a question whether you think it would make sense.